docs/beamrunner/release/1.2.0/troubleshoot.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: docs
+title:  Troubleshooting IBM Streams Runner for Apache Beam
+navtitle: Troubleshooting
+description: You can troubleshoot any issues that occur when you launch, run, or monitor IBM® Streams Runner for Apache Beam applications.
+weight:  10
+published: true
+tag: beam-120
+prev:
+  file: performance
+  title: Performance considerations
+next:
+  file: issues
+  title: Limitations and known issues
+---
+
+You can troubleshoot any issues that occur when you launch, run, or monitor IBM® Streams Runner for Apache Beam applications. Streams Runner includes several pipeline parameters and componentized options to print more information and help narrow the scope to a specific component. 
+
+## Ensure you have the latest fixes
+The latest Streams Runner version is 1.2.0. You can [download](../../../beamrunner-2b-sas/#creating-a-streaming-analytics-service-on-ibm-cloud) the latest Streams Runner installation archive from an IBM Streaming Analytics container-based service. 
+
+## Check known problems and solutions
+See if your problem matches any issues in the [issues and limitations page](../issues.md).
+
+## Enable tracing
+Tracing can give information about the translation and execution of your Beam application and can help debug problems.
+
+### Trace levels
+Streams Runner tracing levels follow SLF4J's levels of `ERROR`, `WARN`, `INFO`, `DEBUG`, and `TRACE` with increasing verbosity and information.
+
+### Translation-time
+Streams Runner uses Beam utilities and the [streamsx.topology](http://ibmstreams.github.io/streamsx.topology/) toolkit to translate your Beam application into a Streams application, submit the application to your IBM Streams instance or IBM Streaming Analytics service, and provide connections to the instance or service for monitoring or cancellation. Log messages about these activites are printed to the console where the Beam program is executed. When specifying the program command, you can use the `--traceTranslation` pipeline option to specify a trace level for the runner, topology toolkit, and/or global scopes. For example, a

```

Comment:
Change:
When specifying the program command, you can use the `--traceTranslation` pipeline option to specify a trace level for the runner, topology toolkit, and/or global scopes.
to:
When you specify the program command, you can use the `--traceTranslation` pipeline option to specify a trace level for the runner, topology toolkit, and global scopes.
Note: Don't use and/or. Pick the one that makes the most sense. If you really need that meaning, rewrite the sentence:
"...for any combination of the runner, topology toolkit, or global scope.
Note: If you use "and", leave "scopes" as is. If you use "or", change "scopes" to "scope".